src/DKIMMailServiceProvider.php

```diff
@@ -35,10 +35,11 @@ private function publishesConfig(): void
     protected function registerIlluminateMailer()
     {
         $this->app->singleton('mail.manager', static function (Application $app) {
-            if (config('dkim.domain') === null) {
+            if (config('dkim.enabled', true) && config('dkim.domain') !== null) {
+                return new MailManager($app);
+            } else {
```

Comment:
this else is useless and can be removed.

Reply:
if you use phpstorm, i recommend this plugin: https://plugins.jetbrains.com/plugin/7622-php-inspections-ea-extended-